common-lib/common/signal/signal.libsonnet

```diff
@@ -135,6 +140,7 @@ local stub = import './stub.libsonnet';
                name=name,
                type=metricType,
                unit=std.get(signalsJson.signals[s], 'unit', ''),
+               nameShort=std.get(signalsJson.signals[s], 'nameShort', name),
```

Comment:
shouldn't it be `nameShort` instead of `name` at the end here?

Reply:
nope, it is `name` as it is default fallback, if nameShort is not found in JSON.

Reply:
Ahh of course that makes more sense, thank you for clarifying!